polaris.shopify.com/content/components/caption/index.md

```diff
@@ -15,10 +15,18 @@ keywords:
   - time text
   - compact text
   - small text
-examples:
-  - fileName: caption-default.tsx
-    title: Default
-    description: Use to provide details in situations where content is compact and space is tight.
+status:
+  value: Deprecated
+  message: This component is no longer supported. Please use the alpha Text component instead.
```

Comment:
Should we just refer to the `Text` component rather than `alpha Text` component?

Reply:
I was thinking of leaving it as `alpha` until we remove the `alpha` flag from the Text component docs. I don't have a super strong opinion on this though, what do you think @aveline?

Reply:
I agree with @aveline because we are starting to remove these old components. Users should be confident in using the new `Text` component.

Reply:
Yes I think referring to `Text` in the docs but leaving the `alpha` tag in the sidebar makes sense.

Reply:
Sounds good, will update thanks for feedback! 👍